zkevm-circuits/src/evm_circuit/execution/end_tx.rs

```diff
@@ -85,14 +93,50 @@ impl<F: Field> ExecutionGadget<F> for EndTxGadget<F> {
         let sub_gas_price_by_base_fee =
             AddWordsGadget::construct(cb, [effective_tip.clone(), base_fee], tx_gas_price);
         let mul_effective_tip_by_gas_used =
-            MulWordByU64Gadget::construct(cb, effective_tip, gas_used);
+            MulWordByU64Gadget::construct(cb, effective_tip, gas_used.clone());
         let coinbase_reward = UpdateBalanceGadget::construct(
             cb,
             coinbase.expr(),
             vec![mul_effective_tip_by_gas_used.product().clone()],
             None,
         );
 
+        // constrain tx receipt fields
+        cb.tx_receipt_lookup(
+            tx_id.expr(),
+            TxReceiptFieldTag::PostStateOrStatus,
+            is_persistent.expr(),
+        );
+        cb.tx_receipt_lookup(
+            tx_id.expr(),
+            TxReceiptFieldTag::LogLength,
+            cb.curr.state.log_id.expr(),
```

Comment:
Ditto, may be this needs to be addressed?

Reply:
as the above !